src/input/password-input.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+
+interface Props {
+    id: string;
+    value: any;
+    onBlur?: (event: React.FormEvent<HTMLInputElement>) => void;
+    append?: string;
+    prepend?: string;
+    onChange: (value: any) => void;
+    disabled?: boolean;
+    className?: string;
+    placeholder?: string;
+    onMouseEnter?: (event: React.MouseEvent<HTMLDivElement>) => void;
+    onMouseLeave?: (event: React.MouseEvent<HTMLDivElement>) => void;
+    autoFocus?: boolean;
+    passwordVisible?: boolean;
+    passwordToggleVisible?: boolean;
```

Comment:
Do we have a name for that feature? I'd prefer sth like `enablePasswordToggle`

Reply:
`enablePasswordToggle` sounds good; will use it.

Reply:
`fixed`